.travis/downstream-environments.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+body='{
+    "request": {
+        "branch":"master",
+        "message":"browser new version: '"$1"'"
```

Comment:
Is this both human-readable and parsable? Could be `Update Browser to 12345`?

Reply:
Human-oriented, will update